tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -663,7 +663,7 @@ function main() {
 		runtime="crio"
 	elif [ "$runtime" == "k3s" ] || [ "$runtime" == "k3s-agent" ] || [ "$runtime" == "rke2-agent" ] || [ "$runtime" == "rke2-server" ]; then
 		containerd_conf_tmpl_file="${containerd_conf_file}.tmpl"
-		if [ ! -f "$containerd_conf_tmpl_file" ]; then
```

Comment:
I don't think this is an issue, sincerely speaking.

So, let's see what's happening here:

In case of k3s or rke2, we do:
* check if the template file exists
* if it does not exist, copy the config file to it

The config file on those distros isalways present.

So, all in all, this does not solve an issue we're seeing.  However, it does make things more robust.

I'd suggest changing the commit message / commit log to expl;ain that, and to match with what it really does.

Reply:
@fidencio when following the operator instructions, for some reason, the source config was not there and I was getting a `/etc/containerd/kata-containers.toml: No such file or directory` on that line.

I'm adapting the commit message to make this clear, tho.

Reply:
It cannot be in that line, as the operator does not support k3s / rke2 stuff.

Reply:
Let me know if you are ok with the commit message, now.